src/AspNet.Security.OAuth.LinkedIn/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+    "version": "1.0.0-alpha1-*",
+    "description": "ASP.NET 5 security middleware enabling LinkedIn authentication.",
+    "authors": [ "Kévin Chalet", "Jerrie Pelser" ],
```

Comment:
Since you're the only author of this great provider, you should probably add just your name :palm_tree: